internal/domain/service/class_service.go

```diff
@@ -41,9 +41,9 @@ func (cs *ClassService) GetClassByCourseID(course_id string) ([]model.Class, err
 	return class, nil
 }
 
-func (cs *ClassService) GetClassBySemester(semester string) ([]uint, error) {
+func (cs *ClassService) GetClassBySemesterAndYear(semester string, year string) ([]uint, error) {
 	var class_id []uint
```

Comment:
This api is using for select class, so I think it would be great if you send back with section. (B.c. user don't know class if)